tests/out/glsl/operators.main.Compute.glsl

```diff
@@ -64,24 +56,156 @@ float constructors() {
     return _e70;
 }
 
-void modulo() {
-    int a_2 = (1 % 1);
-    float b_1 = (1.0 - 1.0 * trunc(1.0 / 1.0));
-    ivec3 c = (ivec3(1) % ivec3(1));
-    vec3 d = (vec3(1.0) - vec3(1.0) * trunc(vec3(1.0) / vec3(1.0)));
+void logical() {
+    bool unnamed_11 = (! true);
```

Comment:
Out of the PR scope, but I think we should remove a redundant extra space after `!` and `-`.

Reply:
Can remove it, the output would be more consistent with the other backends.